components/layout_2020/geom.rs

```diff
@@ -387,6 +396,33 @@ impl<T> physical::Rect<T> {
     }
 }
 
+impl physical::Rect<Length> {
+    pub fn union(&self, other: &Self) -> Self {
```

Comment:
The union of two rectangle is a polygon that is (usually) not a rectangle. This returns the axis-aligned bounding box of that union. Is `aabb` commonly used as an abbreviation in graphics code? How do you feel about renaming? If not, let’s add a doc-comment

Reply:
I was using the nomenclature of euclid, but I'm happy to use `axis_aligned_bounding_box` to describe this concept.